cordova/build.json

```diff
@@ -1,12 +1,15 @@
 {
   "ios": {
     "debug": {
-      "codeSignIdentity": "iOS Distribution",
-      "provisioningProfile": "65f6a621-7665-44e2-b686-93fd24bf4cd5"
+      "codeSignIdentity": "iPhone Developer",
+      "developmentTeam": "6B8FS8W94M",
+      "automaticProvisioning": true
```

Comment:
This won't work as we've 2 provisioning profile on CI, one for staging and one for prod so XCode gets confused between two profiles so we dont need this